main.gms

```diff
@@ -230,548 +217,1220 @@ $setGlobal c_description  REMIND run with default settings
 ***------------------------------------------------------------------------------
 
 ***---------------------    01_macro    -----------------------------------------
+*** (singleSectorGr): 
```

Comment:
Here I would either remove the `:` or write a description

Reply:
I would keep the `:` to remind REMIND developers to add a description :-)